app/components/projects-list/project/template.hbs

```diff
@@ -4,7 +4,18 @@
       {{#text.headline class="h5 project"}}
         {{shared/inline-input value=project.name onChange=(action 'saveProject')}}
       {{/text.headline}}
-      {{#text.subhead}}{{project.version}}{{/text.subhead}}
+      {{#text.subhead}}
+        <small>
+          <div class="layout-row layout-xs-column">
+            <div class="flex">
+              Project: {{project.package.version}}
+            </div>
+            <div class="flex">
+              Ember: {{project.package.devDependencies.ember-cli}}
```

Comment:
Could you make an alias `dependencies` in project model which will concat `devDependencies` and `dependencies`?

Also it would be great to have an alias for version. Then if something change we don't have to worry about it in components.